server/game/gameSystems/DrawSpecificCardSystem.ts

```diff
@@ -86,10 +85,7 @@ export class DrawSpecificCardSystem<TContext extends AbilityContext = AbilityCon
     }
 
     protected override addPropertiesToEvent(event, card: Card, context: TContext, additionalProperties: any = {}): void {
-        const properties = this.generatePropertiesFromContext(context) as IDrawSpecificCardProperties;
         super.addPropertiesToEvent(event, card, context, additionalProperties);
-        // add amount and player to have same properties than drawn event from DrawSystem
-        event.amount = Array.isArray(properties.target) ? properties.target.length : 1;
```

Comment:
I would actually be okay leaving in the `event.amount` logic so that it matches the event emitted by DrawSystem. They both emit "OnCardsDrawn" so the receiving code might not be aware that sometimes `amount` is missing from the event